projects/igniteui-angular/src/lib/grids/grid-base.directive.ts

```diff
@@ -6760,6 +6762,23 @@ export abstract class IgxGridBaseDirective implements GridType,
         });
     }
 
+    /**
+     * @hidden @internal
+     */
+    protected onFocusOut(event: FocusEvent) {
```

Comment:
At this point this can also be reverted since the checks are not needed. But we could refactor the repeated save of `lastActiveNode` and clear of active into a method